examples/build.sh

```diff
@@ -50,6 +50,6 @@ find "$BAL_EXAMPLES_DIR" -type f -name "*.bal" | while read -r BAL_EXAMPLE_FILE;
 done
 
 # Remove generated JAR files
-find "$BAL_HOME_DIR" -type f -name "*.jar" | while read -r JAR_FILE; do
+find "$BAL_HOME_DIR" -maxdepth 1 -type f -name "*.jar" | while read -r JAR_FILE; do
```

Comment:
What's the reason for this limitation?

Reply:
We don't want to remove the jars inside the target. Removing them gives permission error.